### PR TITLE
update [docs] links to a current documentation.

### PR DESCRIPTION
**Pull request by tomekl007**

### CODE REVIEW

File: notebooks/nessie-delta-demo-nba.ipynb [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "Initialize Pyspark\n",
     "----------------------------------------------\n",
     "To get started, we will first have to do a few setup steps that give us everything we need\n",
-    "to get started with Nessie. In case you're interested in the detailed setup steps for Spark, you can check out the [docs](https://projectnessie.org/tools/spark/)\n",
+    "to get started with Nessie. In case you're interested in the detailed setup steps for Spark, you can check out the [docs](https://projectnessie.org/tools/iceberg/spark/)\n",
```
Comment (ajantha-bhat) [inline notebooks/nessie-delta-demo-nba.ipynb:14]: This should be delta's link, not iceberg

https://projectnessie.org/tools/deltalake/spark/

**Review by snazy — APPROVED** [review]

File: notebooks/nessie-iceberg-flink-demo-nba.ipynb [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "Initialize PyFlink\n",
     "----------------------------------------------\n",
     "To get started, we will first have to do a few setup steps that give us everything we need\n",
-    "to get started with Nessie. In case you're interested in the detailed setup steps for Flink, you can check out the [docs](https://projectnessie.org/tools/flink/)\n",
+    "to get started with Nessie. In case you're interested in the detailed setup steps for Flink, you can check out the [docs](https://projectnessie.org/tools/iceberg/flink/)\n",
```
Comment (ajantha-bhat) [inline notebooks/nessie-iceberg-flink-demo-nba.ipynb:14]: In the hive demo, we have useless link in the same place
we need to update it to https://projectnessie.org/tools/iceberg/hive/

Comment (ajantha-bhat) [reply notebooks/nessie-iceberg-flink-demo-nba.ipynb:14]: https://github.com/projectnessie/nessie-demos/blob/main/notebooks/nessie-iceberg-hive-demo-nba.ipynb

Comment (tomekl007) [reply notebooks/nessie-iceberg-flink-demo-nba.ipynb:14]: done :)

**Review by snazy — APPROVED** [review]

**Review by ajantha-bhat — APPROVED** [review]

**Comment by ajantha-bhat** [comment]
@tomekl007 : As the PR is approved, you can merge by yourself once build passes.

**Pull request merged**